### PR TITLE
feat(server): use the lambda handler directly

### PR DESCRIPTION
**Pull request by blacha**
This gives a much better logging output by default

This also enforces pretty printing of logs when run from the console

### CODE REVIEW

File: packages/server/src/server.ts [hunk 1]
```diff
@@ -1,50 +1,40 @@
...
-    ctx.headers.set(HttpHeaderRequestId.RequestId, 'c' + requestId);
-  }
-  const ifNoneMatch = req.header(HttpHeader.IfNoneMatch);
-  if (ifNoneMatch != null && !Array.isArray(ifNoneMatch)) {
-    ctx.headers.set(HttpHeader.IfNoneMatch.toLowerCase(), ifNoneMatch);
-  }
+BasemapsServer.get('/v1/*', async (req: express.Request, res: express.Response) => {
+  const event: ALBEvent = {
+    httpMethod: 'GET',
+    requestContext: { elb: { targetGroupArn: 'arn:fake' } },
```
Comment (Wentao-Kuang) [inline packages/server/src/server.ts:20]: fake Arn?

Comment (blacha) [reply packages/server/src/server.ts:20]: it needed a 'string' for type completeness, doesn't matter what the string is as we dont use it anywhere

Comment (Wentao-Kuang) [reply packages/server/src/server.ts:20]: Weird, this cannot be optional?

Comment (blacha) [reply packages/server/src/server.ts:20]: Nope, this is a AWS ALB event type, so all ALB events should have a arn value there.

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**